docs/guide/ure.md

```diff
@@ -0,0 +1,70 @@
+## Universal Robot UR3e, UR5e and UR10e
+
+%robot ure images/robots/ure/model.png
+
+The Universal Robot [UR3e](https://www.universal-robots.com/products/ur3-robot/), [UR5e](https://www.universal-robots.com/products/ur5-robot/) and [UR10e](https://www.universal-robots.com/products/ur10-robot/) are flexible collaborative robot arms with 6 degrees of freedom.
+
+The Universal Robot UR3e, UR5e and UR10e models in Webots are fully compatible with ROS.
+
+### Movie Presentation
+
+![youtube video](https://www.youtube.com/watch?v=WIY9ebqSXUc)
+
+### UR3e, UR5e and UR10e PROTO
+
+Derived from [Robot](../reference/robot.md).
+
+```
+UR5e/UR5e/UR10e {
+  SFVec3f    translation     0 0 0
+  SFRotation rotation        1 0 0 4.712388966
+  SFString   name            "UR5e"
+  SFString   controller      "void"
+  SFString   controllerArgs  ""
+  SFBool     synchronization TRUE
+  SFBool     selfCollision   TRUE
+  MFNode     toolSlot        []
+  SFBool     staticBase      TRUE
+}
+```
+
+> **File location**: "[WEBOTS\_HOME/projects/robots/universal\_robots/protos/UR3e.proto](https://github.com/omichel/webots/tree/master/projects/robots/universal_robots/protos/UR3e.proto)", "[WEBOTS\_HOME/projects/robots/universal\_robots/protos/UR5e.proto](https://github.com/omichel/webots/tree/master/projects/robots/universal_robots/protos/UR5e.proto)" and "[WEBOTS\_HOME/projects/robots/universal\_robots/protos/UR10e.proto](https://github.com/omichel/webots/tree/master/projects/robots/universal_robots/protos/UR10e.proto)"
+
+#### Field Summary
+
+- `toolSlot`: Extend the robot with new nodes at the end of the arm.
+
+- `staticBase`: Defines if the robot base should be pinned to the static environment.
+
+### Samples
+
+You will find the following sample in this folder: "[WEBOTS\_HOME/projects/robots/universal\_robots/worlds](https://github.com/omichel/webots/tree/master/projects/robots/universal_robots/worlds)".
+
+#### [ure.wbt](https://github.com/omichel/webots/tree/master/projects/robots/universal_robots/worlds/ure.wbt)
+
+![ure.wbt.png](images/robots/ure/ure.wbt.png) This simulation shows an UR3e, an UR5e and an UR10e robots equipped with a [ROBOTIQ 3F Gripper](gripper-actuators.md#robotiq-3f-gripper) grabbing cans on conveyor belts and putting them in crates.
+
+### ROS
+
+To use ROS with the simulated UR5e/UR10e robot in Webots, the `universal_robot_ros` controller should be assigned to the robot.
```

Comment:
We should name it "universal_robots_ros" instead of "universal_robot_ros".

```suggestion
To use ROS with the simulated UR3e/UR5e/UR10e robot in Webots, the `universal_robots_ros` controller should be assigned to the robot.
```